ui/src/app/metadata/detail/metadata-manage/action-bar-vertical/metadata-manage-action-bar-vertical.component.ts

```diff
@@ -72,4 +64,18 @@ export class ManageMetadataActionBarVerticalComponent implements OnInit {
     this.canMetadataCreate = this.authService.isEnabledByRoles(ButtonAction.MetadataCreate);
   }
 
-}
\ No newline at end of file
+  handleDelete(): void {
+    if (confirm("Confirm that you want to delete the Named Reference with name " + this.metadataName)) {
+      this.metadataService.deleteWithResult(this.id).subscribe(data => {
+        if (data && data.success) {
+          this.router.navigate(["/metadata"]).then(
+            () => this.toastService.showToast("Successfully Deleted", "" + this.metadataName)
+          )
+        } else if (data && !data.success) {
+          this.toastService.showToast("Warning", data.message ?? "You cannot delete this one")
```

Comment:
Change text to:
`"Unable to delete metadata - verify this data is not currently associated with a skill"`